types/to-regex/tslint.json

```
@@ -0,0 +1,16 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
+            true,
+            {
+                "mode": "code",
+                "errors": [
+                    ["NoDefaultExport", false],
+                    ["NeedsDefaultExport", false],
+                    ["NeedsExportEquals", false]
+                ]
+            }
+        ]
+    }
+}
```

```suggestion
{ "extends": "dtslint/dt.json" }

```

[Do not add additional rules to tslint.json](https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson)